app/settings/tokens/edit/template.hbs

```diff
@@ -32,19 +32,21 @@
             {{t 'settings.tokens.createSuccess.instructions'}}
         </p>
         <Button
-            {{on 'click' (action this.refresh)}}
+            {{on 'click' (perform this.refreshToken)}}
         >
             {{t 'settings.tokens.createSuccess.editScopes'}}
         </Button>
     {{else}}
         <h4>{{t 'settings.tokens.editToken'}}</h4>
 
-        {{#if this.model.taskInstance.isError}}
-            {{this.model.taskInstance.error}}
-        {{else}}
+        {{#if this.model.taskInstance.isRunning}}
+            <LoadingIndicator @dark={{true}} />
```

Comment:
Very helpful loading indicator here, especially for local dev 🙌

Reply:
Awesome, good to hear.